### ISSUE TITLE
Using Module Locally Fails to Find PostCSS

### ISSUE THREAD

**Issue by Dtphelan1** [namespace-ee/react-calendar-timeline#195]
Has anybody had issues where usage of the package locally fails? I'm using this package in a react-scripts project and I'm getting the following failure when trying to use the module locally: 
```
Error in ../react-calendar-timeline/lib/lib/Timeline.js
Module not found: 'postcss' in project\react-calendar-timeline\lib\lib

 @ ../react-calendar-timeline/lib/lib/Timeline.js 24:0-25
```

That line is `require('./Timeline.css');` and I can't find any reference to postCss in the project except in the autoPrefixer module, and I don't know why that would be called when requiring a css file.

**Comment by maido155** [comment]
I have the same issue, we need help!

**Comment by mcMickJuice** [comment]
Does the solution provided here fix your issue?

https://github.com/namespace-ee/react-calendar-timeline/issues/134#issuecomment-314215244

TL;DR; use `import Timeline from 'react-calendar-timeline/lib'`

**Comment by mcMickJuice** [comment]
Are you using webpack for this project? Can you also include webpack css-loader in your pipeline

https://github.com/webpack-contrib/css-loader

Admittedly, the way this library handles its styling is problematic and will/should be changed

**Comment by Dtphelan1** [comment]
I was building with create-react-app, so yes to webpack; that said, my team is vehemently against ejecting the project and exposing full control over the pipeline. The first solution listed in the referenced issue worked for me! Thanks for responding 👍

**Comment by mcMickJuice** [comment]
Sorry for the trouble. I hope to make the handling of the timelines default CSS file easier to work with going forward.

**Comment by Dtphelan1** [comment]
No worries! Thanks for making such a great library public in the first place! :)
